internal/comby/comby.go

```diff
@@ -0,0 +1,99 @@
+package comby
+
+import (
+	"errors"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os/exec"
+	"strconv"
+	"strings"
+
+	"gopkg.in/inconshreveable/log15.v2"
+)
+
+const combyPath = "comby"
+const numWorkers = 8
+
+func exists() (_ int, err error) {
+	_, err = exec.LookPath("comby")
```

Comment:
I'm assuming you defined `combyPath` above to do this:

```suggestion
	_, err = exec.LookPath(combyPath)
```

Reply:
good catch